### ISSUE TITLE
Static application deployment resource file path is incorrect

### ISSUE THREAD

**Issue by gin280** [nuxt/nuxt.js#3008]
`$ vue init nuxt-community/starter-template <project-name>`,
after setting up the project, run `npm run generate`,resource file path is incorrect

**Comment by qm3ster** [comment]
Yup, the static json api example has been removed from the starter template by now
Good riddance
